janusgraph-hbase-parent/janusgraph-hbase-core/src/main/java/org/janusgraph/diskstorage/hbase/HBaseStoreManager.java

```diff
@@ -275,6 +275,8 @@ public boolean apply(Integer input) {
     private final boolean skipSchemaCheck;
     private final String compatClass;
     private final HBaseCompat compat;
+    // Cached return value of getDeployment() as requesting it can be expensive.
```

Comment:
Will deployment every change over the jvm lifecycle of a janusgraph server backed by hbase? If not, lets make final and precompute.